### ISSUE TITLE
PvC

### ISSUE THREAD

**Issue by phdsg** [VCVRack/community#415]
current:
```
  "name": "PvC",
  "author": "phdsg",
  "license": "BSD-3-Clause",
  "sourceUrl": "https://github.com/phdsg/PvC",
  "latestVersion": "0.6.0",
  "status": "available"
```

**Comment by phdsg** [comment]
let's say, i wanted my plugin listed @ vcvrack.com/plugins with current version info, but not distributed in binary form via the plugin manager.

**Comment by AndrewBelt** [comment]
You can request to remove your repo from the Plugin Manager. The Build Team (me) will remove the `"status": "available"` property.

**Comment by phdsg** [comment]
and for version updates? who changes "latestVersion"?

**Comment by AndrewBelt** [comment]
Then it's "free" and not locked down by the Build Team. You should be able to change it safely.

**Comment by phdsg** [comment]
would it be a bad idea to have that part of the "old system" where devs would submit a PR to change latestVersion. then the teams pick up and change repoVersion if the plugin is in the build cycle and change repoVersion and then buildteam sets distVersion or buildVersion?

**Comment by AndrewBelt** [comment]
Unfortunately that's too complicated. After thinking about this, I complately disagree with my previous message. `"latestVersion"` should be removed for all manifests that are not distributed by the Plugin Manager. Meaning, it should only be present if `"status": "available"`. I will remove this if I remember today.

**Comment by phdsg** [comment]
allright. can i have my "plugin" label now :D

**Comment by AndrewBelt** [comment]
"Plugin" label?

**Comment by phdsg** [comment]
for the issue :)

**Comment by AndrewBelt** [comment]
Ah

**Comment by phdsg** [comment]
thx.
so to sum:
if a dev requests removal from the manager but still wants to be listed on the plugins page,
we remove the repo and submodule and the manifest gets stripped of the latestVersion and status?

**Comment by AndrewBelt** [comment]
Build Team will handle remove of the actual build. Yes, removing `lastestVersion` and `status`, and also removing the packages from the server.

**Comment by phdsg** [comment]
i disabled the git-pages website (`pluginUrl`).
see [first post](https://github.com/VCVRack/community/issues/415#issue-310349841) for requested changes.

**Comment by AndrewBelt** [comment]
Fixed with 54d9ab0

**Comment by phdsg** [comment]
thx.
